Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -61,14 +61,16 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   // The sforms should be identical in both cases.
   if (LPSLabel_direction != LPSLabel_noqform_direction)
   {
+    std::cerr << "Error: sform direction matrices differ when qform is not present" << std::endl;
```

Comment:
In order to convey a more clear information about the error rationale, I would rephrase this as:
```
std::cerr << "Error: direction data must be identical for qfrom- and sform-complete and sform-only versions of the same image." << std::endl;
```

@cookpa Do you think it is worthwhile adding some further information about the failure ? Something like, e.g.
```
std::cerr << "qfrom- and sform-complete image direction: " << LPSLabel_direction << std::endl;
std::cerr << "sform-only image direction:" << LPSLabel_noqform_direction << std::endl;
```

Reply:
Addressed in 7f16366.